src/styles/blocks/utils/variables.scss

```
@@ -0,0 +1,11 @@
+$cell-size: 100px;
+$cell-gap: 1px;
+$cell-width: 7 * ($cell-size + $cell-gap);
+
+
+$transition-cell: color 0.5s, transform 0.5s;
```

this doesn't work right now
```suggestion
$transition-cell: background-color 0.5s, transform 0.5s;
```